.github/workflows/python-app-py3.yml

```diff
@@ -11,24 +11,28 @@ on:
 
 jobs:
   build:
-
+    strategy:
+      matrix:
+        python-version: [3.6, 3.7]
```

Comment:
Add in your description about what this is doing?

Reply:
done.